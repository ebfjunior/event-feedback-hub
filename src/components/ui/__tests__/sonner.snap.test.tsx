import React from 'react';
import { render } from '@testing-library/react';
import { Toaster } from '../sonner';

vi.mock('next-themes', () => ({ useTheme: () => ({ theme: 'light' }) }));
vi.mock('sonner', () => ({ Toaster: (props: any) => <div data-mock="sonner" {...props} /> }));

describe('ui/Sonner Toaster (snapshot)', () => {
	it('renders', () => {
		const { container } = render(<Toaster />);
		expect(container.firstChild).toMatchSnapshot();
	});
});


