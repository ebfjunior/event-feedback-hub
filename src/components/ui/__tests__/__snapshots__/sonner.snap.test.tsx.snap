// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ui/Sonner Toaster (snapshot) > renders 1`] = `
<div
  class="toaster group"
  data-mock="sonner"
  style="--normal-bg: var(--popover); --normal-text: var(--popover-foreground); --normal-border: var(--border);"
  theme="light"
/>
`;
